src/Exceptions/InvalidValueException.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\Number\Exceptions;
+
+use InvalidArgumentException;
+
+class InvalidValueException extends InvalidArgumentException implements NumberException
```

Comment:
Is this class now redundant?

Reply:
This is used in the StrictPercentage class. 👍🏻